app/assets/javascripts/application.js

```diff
@@ -13,6 +13,8 @@
 //= require jquery_ujs
 //= require dsds
 //= require date_chooser
+//= require show_hide
+//= require disable_checker
```

Comment:
Should we require these in the files they're needed in? Instead of having all dependencies specified in the main file? I'm just asking, I don't know if there are benefits/requirements of having it here.